### PR TITLE
Bookstore: styling

### PR DESCRIPTION
**Pull request by lincoln1883**
For this milestone i was able to accumplish the following tasks:
- Use the CSS rules detailed in Zeplin to style the website.
- Make the final design match Zeplin's design.
- I made: (this includes all elements + the same styling). and im sorry.
- Add missing JSX elements to your project .
- Im very sorry (without adding extra functionality).

### CODE REVIEW

File: src/components/Navbar.jsx [hunk 1]
```diff
@@ -2,17 +2,17 @@ import React from 'react';
...
+      <ul className="flex justify-center gap-2 mt-1">
         <li className="nav-link">
-          <NavLink className="nav-item" to="/books">BOOKS</NavLink>
+          <NavLink className=" text-sm text-black hover:text-gray-600 active:bg-gray-200" to="/books">BOOKS</NavLink>
         </li>
         <li className="nav-link">
-          <NavLink className="nav-item" to="/categories">CATEGORIES</NavLink>
+          <NavLink className=" text-sm  text-black hover:text-gray-600 active:bg-gray-200" to="/categories">CATEGORIES</NavLink>
         </li>
       </ul>
```
Comment (thecodechaser) [inline src/components/Navbar.jsx]: - [ ] Please add the user icon on the right corner in the header to match the Zepline design.

### Your design

image

### Zepline design

image

**Review by thecodechaser — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ 

## Hi @lincoln1883👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 🎉
- Nice UI design. ✔️
- Gitflow is followed. :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- No linter errors. :heavy_check_mark:
- PR has a good title and summary. ✔️



## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Navbar.jsx (hunk 1 above)
Comment (lincoln1883) [reply src/components/Navbar.jsx]: Done thank you.

**Review by roshan-bajgain — APPROVED** [review]
Hi @lincoln1883, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- GitHub flow is followed. ✔️
- PR has a good title and summary. ✔️
- Good job on implementing the changes ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**